tutorial/4. multiple_grid_cells_and_hypercube.ipynb

```diff
@@ -0,0 +1,1231 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "display(df)\n",
+    "df.plot(x='time.s', y=['CONC.A.mol m-3', 'CONC.B.mol m-3', 'CONC.C.mol m-3'], title='Concentration over time', ylabel='Concentration (mol m-3)', xlabel='Time (s)')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "50fb5839",
+   "metadata": {},
+   "source": [
+    "# Latin Hypercube Sampling in MUSICA\n",
+    "Latin Hypercube Sampling (LHS) is a statistical method for generating multidimensional random samples.</br>\n",
+    "It avoids the problem of clustering that can sometimes appear in pure random sampling.</br>\n",
+    "This tutorial will go over a simple example of utilizing LHS to run a multi-grid-cell solver in MUSICA.</br>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "78624def",
+   "metadata": {},
+   "source": [
+    "## 1. Rerunning Previous Code\n",
+    "This is simply a copy of the first 4 steps from the above multiple grid cells tutorial since the code is identical up to this point."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Why are we redefining the system? If this hasn't changed, let's not add redundant code